src/test/java/com/example/produktapi/ProductTests.java

```diff
@@ -1,15 +1,28 @@
 package com.example.produktapi;
 
 import com.example.produktapi.model.Product;
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
 public class ProductTests {
 
-    Product a =new Product("SchoolBag",123.222,"Bag","Adidas","img.jpg");
-@Test
-public void checkProductTitle(){
-
-    System.out.println(a.getTitle());
+    Product product =new Product("Bracelet",123.222,"Jewelery","Silver dragon bracelet","img.jpg");//
+    // Suzana. Testing Get Description. Positive testing.
+    @Test
+public void checkProductDescription(){
+    Assertions.assertEquals("Silver dragon bracelet", product.getDescription());
 
 }
+    // Suzana. Testing Get Description. Negative testing.
```

Comment:
Running this testcase will result in test failure as Actual and expected description are not equal. Please user assertNotEquals instead of assertEquals .